packages/rocketchat-lib/client/lib/RestApiClient.js

```diff
@@ -70,7 +70,7 @@ RocketChat.API = {
 
 		return new Promise(function _jqueryFormDataPromise(resolve, reject) {
 			jQuery.ajax({
-				url: `${ Meteor.absoluteUrl() }api/${ endpoint }${ query }`,
+				url: `${ window.location.origin }/api/${ endpoint }${ query }`,
```

Comment:
why not use? 

```
url: Meteor.absoluteUrl(`${ /api/${ endpoint }${ query }`),
```

Reply:
Because the `Meteor.absoluteUrl` uses the configured url and when you are testing against a specific instance, it calls another instance. We currently don't use these utilities, so we can improve them later.

Reply:
oh.. we don't take care of a case like this in anywhere.. we always assume all instances are at same domain.